truenas/webui
# NAS-110543 / 21.08 / Added 'name_regex' to ReplicationWizard and ReplicationForm

NAS-110543

To test

- Add a new replication task with remote source dataset
- You should be able to either edit 'naming_schema' (& 'also_include_naming_schema') or 'name_regex'. They shouldn't be editable at the same time. Either or.
- Check that the form submission is sending the correct payload to the API and replication task is created successfully.
